### PR TITLE
#2039 - add storage keyword highlighting in book

### LINKED ISSUE #2039: `storage` keyword highlighting is missing from the `Sway` book
https://fuellabs.github.io/sway/master/blockchain-development/storage.html

### PR DESCRIPTION
**Pull request by r-sitko**
close #2039

### CODE REVIEW

**Review by mohammadfawaz — DISMISSED** [review]

File: docs/theme/highlight.js [hunk 1]
```diff
@@ -303,7 +303,7 @@ anyNumberOfTimes:g};for(const e in A)"object"==typeof A[e]&&n(A[e])
 grmr_sway:e=>{const t={className:"title.function.invoke",relevance:0,
 begin:b(/\b/,/(?!let\b)/,e.IDENT_RE,u(/\s*\(/))},n="([u](8|16|32|64))?";return{
 name:"Sway",aliases:["sw"],keywords:{$pattern:e.IDENT_RE+"!?",
-keyword:["abi","as","asm","const","contract","deref","enum","fn","if","impl","let","library","match","mut","else","predicate","ref","return","script","Self","self","str","struct","trait","use","where","while"],
+keyword:["abi","as","asm","const","contract","deref","enum","fn","if","impl","let","library","match","mut","else","predicate","ref","return","script","Self","self","str","struct","trait","use","where","while","storage"],
```
Comment (adlerjohn) [inline docs/theme/highlight.js:306]: Alphabetize
```suggestion
keyword:["abi","as","asm","const","contract","deref","enum","fn","if","impl","let","library","match","mut","else","predicate","ref","return","script","Self","self","storage","str","struct","trait","use","where","while"],
```

Comment (adlerjohn) [reply docs/theme/highlight.js:306]: Ah actually this file is auto-generated.

File: scripts/highlightjs/sway.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ export default function(hljs) {
     "use",
     "where",
     "while",
+    "storage"
```
[the remark this answers is not in the document]
Comment (r-sitko) [reply scripts/highlightjs/sway.js]: Done

File: docs/theme/highlight.js (hunk 1 above)
Comment (r-sitko) [reply docs/theme/highlight.js]: Yeah, it's auto generated by scripts/highlightjs/build.sh script.

**Review by adlerjohn — APPROVED** [review]

**Review by mohammadfawaz — APPROVED** [review]

**Pull request merged**